app/models/dtos.py

```diff
@@ -0,0 +1,35 @@
+from enum import Enum
+from pydantic import BaseModel
+from datetime import datetime, timezone
+
+
+class LLMModel(str, Enum):
+    GPT35_TURBO = "gpt-3.5-turbo"
```

Comment:
This does not match the enum in Artemis:
```suggestion
    GPT35_TURBO = "GPT35_TURBO"
```